week5/repdigit/lib/repdigit.rb

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env ruby
-$LOAD_PATH << './lib'
+$LOAD_PATH << '../lib'
```

Comment:
now works every time you call this file directly or require it in another file

Reply:
ok